meta-balena-common/recipes-bsp/grub/grub-conf/grub.cfg_internal_template

```diff
@@ -0,0 +1,69 @@
+# Automatically created by OE
+serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+default=boot
+insmod ext2
+insmod gzio
```

Comment:
What is this module used for?

Reply:
@floion A gzipped kernel image, such as when `KERNEL_IMAGETYPE="Image.gz"`

Reply:
@jakogut is this still used? Most if not all would use bzImage which is self-extracting anyway so we would not need this grub module for that.
I thought the module might be used for something else.

Reply:
@floion Image.gz is still useful on architectures that don't support self-extracting kernel images, such as arm64.

For reference, here's the relevant section from the kernel's `make help`:
```
$ >> ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- make help
<snip>
Architecture specific targets (arm64):
* Image.gz      - Compressed kernel image (arch/arm64/boot/Image.gz)
  Image         - Uncompressed kernel image (arch/arm64/boot/Image)
  install       - Install uncompressed kernel
  zinstall      - Install compressed kernel
                  Install using (your) ~/bin/installkernel or
                  (distribution) /sbin/installkernel or
                  install to $(INSTALL_PATH) and run lilo

  bcm2711_defconfig        - Build for bcm2711
  bcmrpi3_defconfig        - Build for bcmrpi3

  make V=0|1 [targets] 0 => quiet build (default), 1 => verbose build
  make V=2   [targets] 2 => give reason for rebuild of target
  make O=dir [targets] Locate all output files in "dir", including .config
  make C=1   [targets] Check re-compiled c source with $CHECK (sparse by default)
  make C=2   [targets] Force check of all c source with $CHECK
  make RECORDMCOUNT_WARN=1 [targets] Warn about ignored mcount sections
  make W=n   [targets] Enable extra build checks, n=1,2,3 where
		1: warnings which may be relevant and do not occur too often
		2: warnings which occur quite often but may still be relevant
		3: more obscure warnings, can most likely be ignored
		Multiple levels can be combined with W=12 or W=123

Execute "make" or "make all" to build all targets marked with [*] 
For further info see the ./README file
```